Documentation/Contributors/CLAs/corporate-cla-agi-v1.0.txt

```diff
@@ -0,0 +1,147 @@
+                      Analytical Graphics, Inc.
+  Software Grant and Corporate Contributor License Agreement ("Agreement")
+           http://www.agi.com/licenses/corporate-cla-agi-v1.0.txt
```

Comment:
We are evaluating lots of CLA ideas so please hold on the new repo, just removing the link would be OK.